ckanext/fcscopendata/logic/action.py

```diff
@@ -57,6 +60,18 @@ def package_create(up_func, context, data_dict):
     data_dict['title'] =  data_dict.get('title_translated-en', '')
     data_dict['notes'] =  data_dict.get('notes_translated-en', '')
 
+    start_period = str(data_dict.get('start_period')) + "-01"
+    end_period = str(data_dict.get('end_period')) + "-01"
```

Comment:
Does` 01` refers to days?  if that so,
 I think the end day should be the last day of the particular month. 
 eg. if selected April as the end month then the day should be 29.